keyboards/converter/adb_usb/rp2040/config.h

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2011 Jun Wako <wakojun@gmail.com>
+Copyright 2025 Noah Patel <ImNoahDev@imnoah.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+RP2040 port for RP2040-Zero board
+*/
+
+#pragma once
+
+/* ADB port setting for RP2040 */
+#define ADB_DATA_PIN GP15
+
+/* RP2040 double-tap reset bootloader configuration */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
```

Comment:
```suggestion
```

The timeout isn't needed. If the double tab reset is defined, then it defaults to 200U